mesa/space.py

```diff
@@ -1323,6 +1361,16 @@ def __init__(
         self.size = np.array((self.width, self.height))
         self.torus = torus
 
+        # Store random number generator
```

Comment:
this can be included in the property setter if you built on my suggestion. Also means you don't have to repeat it in all classes but can keep it only in the root class.